packages/driver/src/cypress/command.ts

```diff
@@ -72,15 +72,14 @@ export class $Command {
       } else {
         log.set('snapshot', false)
         log.finish()
+        if (Cypress.isCrossOriginSpecBridge) {
+          log.fireChangeEvent.flush()
+        }
       }
     })
   }
 
   log (log) {
-    // always set the chainerId of the log to ourselves
-    // so it can be queried on later
-    log.set('chainerId', this.get('chainerId'))
```

Comment:
Why is this no longer needed?

Reply:
This is handled by the log instance now to prevent an immediate log change updated when a log is created and added to the command: https://github.com/cypress-io/cypress/pull/28531/files/b8778c05a699995e4ca3e4c372dc2566dd26256f#diff-e8daebc8b7c0a8dfa8c7119183dcd9e9e44e0d7a2ff886808489b57dd231dcdaR194